misc/atrium-specs/src/commonMain/kotlin/ch/tutteli/atrium/specs/integration/AbstractIterableNotToHaveElementsOrNoneExpectationsTest.kt

```diff
@@ -0,0 +1,140 @@
+        notToHaveElementsOrNoneSpec,
+        notToHaveElementsOrNoneNullableSpec
+    )
+    { notToHaveElementsOrNoneFun ->
+        it("does not throw") {
+            expect(emptyIterable()).notToHaveElementsOrNoneFun { toBeLessThan(1.1) }
+        }
+    }
+
+    @TestFactory
+    fun one_to_7__does_not_throw() = testFactory(notToHaveElementsOrNoneSpec)
+    { notToHaveElementsOrNoneFun ->
+        listOf(1.2, 2.2, 3.3).forEach {
+            it("$toEqualDescr($it) does not throw") {
+                expect(oneToSeven()).notToHaveElementsOrNoneFun { toEqual(it) }
+            }
+        }
+    }
+
+
+    @TestFactory
+    fun one_to_7__throws() = testFactory(
+        notToHaveElementsOrNoneSpec
+    ) { notToHaveElementsOrNoneFun ->
+        it("$toEqualDescr(4.1) throws AssertionError") {
+            expect {
+                expect(oneToSeven()).notToHaveElementsOrNoneFun { toEqual(4.1) }
+            }.toThrow<AssertionError> {
+                message {
+                    toContainRegex(
+                        "\\Q$rootBulletPoint\\E$notToHaveElementsOrNoneDescr: $separator" +
+                            "$indentRootBulletPoint$indentListBulletPoint\\Q$explanatoryBulletPoint\\E$toEqualDescr: 4.1.*$separator" +
+                            "$indentRootBulletPoint$indentListBulletPoint\\Q$warningBulletPoint$mismatches:\\E $separator" +
+                            "${mismatch(2, "4.1")}.*$separator" +
+                            "${mismatch(3, "4.1")}.*$separator" +
+                            "${mismatch(8, "4.1")}.*"
+                    )
+                }
+            }
+        }
+    }
+
+    @TestFactory
+    fun nullable_cases() = testFactory(notToHaveElementsOrNoneNullableSpec) { notToHaveElementsOrNoneFun ->
+        describe("iterable ${oneToSeven().toList()}") {
+            it("null does not throw") {
+                expect(oneToSeven() as Iterable<Double?>).notToHaveElementsOrNoneFun(null)
+            }
+        }
+        describe("iterable ${oneToSevenNullable().toList()}") {
+            it("null throws AssertionError") {
+                expect {
+                    expect(oneToSevenNullable()).notToHaveElementsOrNoneFun(null)
+                }.toThrow<AssertionError> {
+                    message {
+                        toContainRegex(
+                            "\\Q$rootBulletPoint\\E$notToHaveElementsOrNoneDescr: $separator" +
+                                "$indentRootBulletPoint$indentListBulletPoint\\Q$explanatoryBulletPoint\\E$toEqualDescr: null$separator" +
+                                "$indentRootBulletPoint$indentListBulletPoint\\Q$warningBulletPoint$mismatches:\\E $separator" +
+                                "${mismatch(1, "null")}.*$separator" +
+                                "${mismatch(5, "null")}.*"
+                        )
+                    }
+                }
+            }
+        }
+        it("1.1 throws AssertionError") {
```

Comment:
you closed it too early, should be after the `it` block
```suggestion
        it("1.1 throws AssertionError") {
```

Reply:
@robstoll - Done